README.md

```diff
@@ -186,16 +182,16 @@ Set the `PLATFORM` variable when run `make`:
 
 ```bash
 # Build helloworld for raspi4
-make PLATFORM=aarch64-raspi4 A=examples/helloworld
+make PLATFORM=aarch64-raspi4 SMP=4 A=examples/helloworld
 ```
 
 You may also need to select the corrsponding device drivers by setting the `FEATURES` variable:
 
 ```bash
 # Build the shell app for raspi4, and use the SD card driver
-make PLATFORM=aarch64-raspi4 A=examples/shell FEATURES=driver-bcm2835-sdhci
+make PLATFORM=aarch64-raspi4 SMP=4 A=examples/shell FEATURES=page-alloc-4g,driver-bcm2835-sdhci BUS=mmio
```

Comment:
if we do not set `FEATURES=page-alloc-4g`, the default allocator max size will be 256M, right?

Reply:
Yes, and it will panic on raspi4 platform.